File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task125_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <stdio.h>
#include <string.h>

void accessResource(char resource[]) {
    if (strcmp(resource, "admin") == 0) {
        printf("Access granted to admin resource
");
    } else if (strcmp(resource, "moderator") == 0) {
        printf("Access granted to moderator resource
");
    } else if (strcmp(resource, "user") == 0) {
        printf("Access granted to user resource
");
    } else {
        printf("Access denied
");
    }
}

int main() {
    char resource[20];
    printf("Enter the resource you want to access: ");
    scanf("%19s", resource);
    accessResource(resource);
    return 0;
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task125_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the resource you want to access: ");
        String resource = scanner.nextLine();
        accessResource(resource);
    }

    public static void accessResource(String resource) {
        if (resource.equals("admin")) {
            System.out.println("Access granted to admin resource");
        } else if (resource.equals("moderator")) {
            System.out.println("Access granted to moderator resource");
        } else if (resource.equals("user")) {
            System.out.println("Access granted to user resource");
        } else {
            System.out.println("Access denied");
        }
    }
}